### DISCUSSION TITLE
Timestamp in syslog file

### DISCUSSION THREAD

**Discussion by daemonlover7** [datalust/seq-tickets#2070] [Help]
Hello, i have newly installed seq and setup syslog module. i added only one mikrotik router ti log. it's logging the data and showing with my timezone timestamp accurately, but when it comes to exporting logs the timestamp is different. is there anything i need to change/configure when i want to export logs.
![Screenshot 2024-01-24 110111](https://github.com/datalust/seq-tickets/assets/97615064/93043aae-d734-48ba-b9fe-324aca211de4)
![Screenshot 2024-01-24 110133](https://github.com/datalust/seq-tickets/assets/97615064/e6004e38-852f-43ab-850f-afa892512705)

**Comment by KodrAus** [comment]
Hi @daemonlover7 :wave:

The default CSV export will use UTC timestamps. You can convert those back into your local timezone to export using a query, and export the results. For example, I can get an export of timestamps and messages in AEST with a query like this:

```
select ToIsoString(@Timestamp, OffsetIn('Australia/Brisbane', @Timestamp)) as timestamp, @Level as level, @Message as message from stream limit 10
```

You can tweak the query to add other properties or increase the limit as needed.

Hope that helps!
